tubemq-core/src/main/java/org/apache/tubemq/corebase/utils/AddressUtils.java

```diff
@@ -34,31 +39,33 @@ public static synchronized String getLocalAddress() throws Exception {
         return localIPAddress;
     }
 
-    public static boolean validLocalIp(String currLocalHost) throws Exception {
+    public static synchronized void setLocalAddress(String localIPAddress) {
+        AddressUtils.localIPAddress = localIPAddress;
+    }
+
+    public static boolean validLocalIp(String currLocalHost) {
```

Comment:
The validLocalIp function seems to be superfluous and should be removed along with the sub-functions